packages/web/component/OrderHistory.tsx

```diff
@@ -0,0 +1,75 @@
+import { ExternalLinkIcon } from "@chakra-ui/icons";
+import { Box, Divider, Heading, Link, Table, TableContainer, Tbody, Td, Th, Thead, Tr } from "@chakra-ui/react";
+import { Web3Provider } from "@ethersproject/providers";
+import { useWeb3React } from "@web3-react/core";
+
+import { getNetworkFromChainId } from "../lib/web3";
+
+export const OrderHistory: React.FC = () => {
+  const { account } = useWeb3React<Web3Provider>();
+
+  const getTxHistory = () => {
+    let array = [];
+    if (account && typeof window !== "undefined") {
+      if (window.localStorage) {
+        const pastJson = localStorage.getItem(account);
+        if (pastJson) {
+          array = JSON.parse(pastJson);
+        }
+      }
+    }
+    return array;
+  };
+
+  const data = getTxHistory();
+
+  console.log(account);
+
+  return (
+    <Box>
+      {account && (
+        <Box mt="10px">
+          <Divider mb={12} />
+          <Heading fontSize={{ base: "xl", md: "3xl" }} px="2px">
+            Bridge History
+          </Heading>
+          <TableContainer>
+            <Table size="sm">
+              <Thead>
+                <Tr textAlign={"center"}>
+                  <Th>Index</Th>
+                  <Th>Transaction Hash</Th>
+                  <Th>Connextscan</Th>
+                </Tr>
+              </Thead>
+              <Tbody>
+                {data.map((d: [string, string, number], index: number) => {
+                  const chainName = getNetworkFromChainId(d[2]);
+                  const chaineExplorer = "https://" + chainName + ".etherscan.io/tx/" + d[0];
```

Comment:
I think you should get explorer from here 
https://github.com/nfthashi/monorepo/blob/main/packages/contracts/networks.json

Reply:
Certainly I should do! Thanks